openwisp_network_topology/templates/netjsongraph/netjsongraph-script.html

```diff
@@ -190,7 +190,7 @@
             const data=JSON.parse(e.data);
             if(data.type==="broadcast_topology"){
                 const topology=JSON.parse(data.topology);
-                window.__njg_graph__.utils.JSONDataUpdate.call(window.__njg_graph__, topology);
+                window.graph.utils.JSONDataUpdate.call(window.graph, topology);
```

Comment:
@nemesisdesign I see that you have made this change in https://github.com/openwisp/openwisp-network-topology/commit/e498409e72e81dae6190aab5463e5fed09fcdbd5#diff-0146282d64adc7cf5c7655b478b500c652a2a955faee88caf2558fac3c06c609R164-R165. Was it unintentional?

Reply:
Thanks for spotting this, I must have forgot to test this and since we don't have an automated test for this I didn't get any failure.